apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -282,6 +354,18 @@ static void tc_task_task_restart(void)
 	int ret_chk;
 	unsigned int remain;
 	g_callback = 0;
+
+	/* Check for NULL pid parameter  */
+
+	pid = task_create("tc_exit", SCHED_PRIORITY_MAX - 1, 1024, restart_task, (char * const *)NULL);
+	TC_ASSERT_GT("task_create", pid, 0);
+
```

Comment:
1. Let's check g_callback value to confirm that task is not restarted.
```
	pid = task_create("tc_exit", SCHED_PRIORITY_MAX - 1, 1024, restart_task, (char * const *)NULL);
	TC_ASSERT_GT("task_create", pid, 0);

	g_callback = 0;
	ret_chk = task_restart(0);
	TC_ASSERT_EQ("task_restart", ret_chk, ERROR);
	TC_ASSERT_EQ("task_restart", errno, ENOSYS);
	TC_ASSERT_EQ("task_restart", g_callback, 0);
```
2. If you added ```g_callback = 0; like above, you should remove line 356. It is unnecessary.
3. After test this, you should reinitialize g_callback to test next.